docs/product-user-guide/src/main/asciidoc/business-central-settings-security-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='business-central-settings-security-proc']
+= Security settings
+
+To access the Security perspective, click *Roles*, *Groups*, or *Users* on the Settings perspective. Depending on which icon you click, the Security perspective opens on the Roles, Groups, or Users tab. Use roles and groups to manage privileges and settings for users. Any changes apply to all users with the affected role or group.
```

Comment:
Should also be bold in "opens in *Roles*, *Groups*, or *Users* tabs." Our internal style to date has been to bold all UI elements, whether active or not. From what I understand.